ArduCopter/mode_guided.cpp

```diff
@@ -420,7 +420,8 @@ void ModeGuided::pos_control_run()
 
     // if not armed set throttle to zero and exit immediately
     if (is_disarmed_or_landed()) {
-        make_safe_spool_down();
+        // do not spool down tradhel when on the ground with motor interlock enabled
+        make_safe_ground_handling(!copter.is_tradheli());
```

Comment:
I don't think we should need these changes to guided mode.  The vehicle should only get into this "disarmed_or_landed" state if it has landed for some reason which is extremely rare.  Once it has landed the user should trigger a takeoff using the Guided_takeoff controller.